docs/CLI.md

```diff
@@ -61,3 +61,12 @@ gulp has very few flags to know about. All other flags are for tasks to use if n
 
 **--log-level**, **-L**
     Set the loglevel. -L for least verbose and -LLLL for most verbose. -LLL is default.
+
+**--config**, **-c**
```

Comment:
Not sure about having this option either.  I was expecting the config files to be static (always named `.gulp.{ext}`).  Why do we want to make this configurable?